### DISCUSSION TITLE
Could you please add a compressed package preview function? Thank you.

### DISCUSSION THREAD

**Discussion by a0n1** [aonez/Keka#1387] [Ideas]
Please, please add a preview function so that we can see the files inside the compressed package without extracting them. It would be great to be able to choose to extract only specific files instead of having to extract everything every time. Thank you.

**Comment by gingerbeardman** [comment]
Dupe of #100 

It's coming "soon"

**Reply by a0n1** [reply]
🥲

**Comment by DanielQu9** [comment]
To this day, we still don't know when he will complete the preview feature.😭

**Reply by gingerbeardman** [reply]
It's already in the iOS version, so "soon" is a good bet

**Reply by bigplayer-ai** [reply]
> It's already in the iOS version, so "soon" is a good bet

It’s listed under [Things to come, not in the near future](https://github.com/aonez/Keka/milestone/4?page=2), so “soon” seems unlikely.

Hopefully, Can Quick Look support can be added separately from [See contents of files](https://github.com/aonez/Keka/issues/100)? which seems to be delayed due to complexity. 
Tools like [qlZipInfo](https://github.com/srirangav/qlZipInfo) which [stopped working on macOS sequoia 15.x](https://github.com/srirangav/qlZipInfo#:~:text=6.%20MacOS%20X%20Sequoia%20(15.x)%20and%20newer%20are%20not%20supported%20%20%20%20%20%20%20because%20they%20no%20longer%20support%20the%20Quicklook%20generator%20%20%20%20%20%20%20framework.%20%20See:%20%20%20%20%20%20%20https://eclecticlight.co/2024/10/31/how%2Dsequoia%2Dhas%2Dchanged%2Dquicklook%2Dand%2Dits%2Dthumbnails/).

**Comment by gcobr** [comment]
This would be a critical feature for me. Very few tools in the market provide it on Mac.

**Comment by suxiaolei97** [comment]
Today is August 3, 2026. Browsing files without extracting them is still not available in the macOS version. I hope that AI agents can assist the developers in solving these issues and finally bring this feature to Keka.
